drivers/led/lp5569.c

```diff
@@ -30,12 +31,15 @@ LOG_MODULE_REGISTER(lp5569, CONFIG_LED_LOG_LEVEL);
 
 #define LP5569_MISC			0x2F
 #define LP5569_POWERSAVE_EN		BIT(5)
+#define LP5569_CP_MODE_SHIFT		3
 
 /* PWM base Register for controlling the duty-cycle */
 #define LP5569_LED0_PWM			0x16
 
 struct lp5569_config {
 	struct i2c_dt_spec bus;
+	struct gpio_dt_spec en_pin;
```

Comment:
Usually follows DTS property name
```suggestion
	struct gpio_dt_spec enable_gpios;
```

Reply:
Liberty taken to keep it singular in the code. Multiplum in the dts for gpio core parsing reasons.